webpack/js/app.js

```diff
@@ -5,17 +5,16 @@ import "bootstrap";
 function initDownloadButton() {
   const downloadButton = document.querySelector(".download-btn");
   if (downloadButton) {
-    fetch("https://api.github.com/repos/lektor/lektor/releases", {
-      method: "GET",
-    })
+    // See https://warehouse.pypa.io/api-reference/json.html#project
+    fetch("https://pypi.org/pypi/lektor/json", { method: "GET" })
       .then((res) => res.json())
-      .then((releases) => {
-        const tag = releases[0].tag_name;
+      .then((project) => {
+        const version = project.info.version;
         const link = downloadButton.querySelector("a");
         if (link) {
```

Comment:
Nit: This conditional could be moved outside the `fetch`.  I realize it is not an expected case, but there's no point in making the fetch request if we might decide not to use the result anyway.

E.g.
```js
const downloadLink = document.querySelector(".download-btn a")
if (downloadLink) {
    fetch(...
```

Reply:
:+1: - this is from the straight refactor from the more involved old code (#359) where I didn't realize that we don't need the button element itself anymore.